app/src/models/statistic.js

```diff
@@ -4,6 +4,9 @@ const { Schema } = mongoose;
 
 const Statistic = new Schema({
     slug: { type: String, required: true, trim: true },
+    application: {
+        type: String, required: true, trim: true, default: 'gfw'
```

Comment:
not sure I agree with having a default. I know that has been kind of "standard policy" so far, but I feel like this does more harm than good

Reply:
As discussed in a previous meeting (and noted [here](https://www.pivotaltracker.com/story/show/170311809)), removed `gfw` as default value for *both* statistics and subscriptions.